modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -0,0 +1,139 @@
+control the label field of the Kubernetes plug-in pod template by setting an
+annotation on the imagestream or imagestream tag object with the key
+`slave-label`. Otherwise, the name is used as the label.
+
+[NOTE]
+====
+If you log into the Jenkins console and make further changes to the Pod Template
+configuration after the Pod Template is created, and the OpenShift Sync plug-in
+detects that the image associated with the ImageStream or ImageStreamTag has
+changed, it will replace the Pod Template and overwrite those configuration
+changes. No merging of configuration will occur.
+
+Consider the ConfigMap approach if you have more complex configuration needs.
+====
+
+When it finds a ConfigMap with the appropriate label, it assumes that any
+values in the key-value data payload of the ConfigMap contains XML consistent
+with the config format for Jenkins and the Kubernetes plug-in pod templates. A
+key differentiator to note when using ConfigMaps, instead of imagestreams or
+imagestream tags, is that you can control all the various fields of the
+Kubernetes plug-in pod template.
+
+The following is an example ConfigMap:
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: jenkins-agent
+  labels:
+    role: jenkins-slave
+data:
+  template1: |-
+    <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
+      <inheritFrom></inheritFrom>
+      <name>template1</name>
+      <instanceCap>2147483647</instanceCap>
+      <idleMinutes>0</idleMinutes>
+      <label>template1</label>
+      <serviceAccount>jenkins</serviceAccount>
+      <nodeSelector></nodeSelector>
+      <volumes/>
+      <containers>
+        <org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
+          <name>jnlp</name>
+          <image>openshift/jenkins-agent-maven-35-centos7:v3.10</image>
+          <privileged>false</privileged>
+          <alwaysPullImage>true</alwaysPullImage>
+          <workingDir>/tmp</workingDir>
+          <command></command>
+          <args>${computer.jnlpmac} ${computer.name}</args>
+          <ttyEnabled>false</ttyEnabled>
+          <resourceRequestCpu></resourceRequestCpu>
+          <resourceRequestMemory></resourceRequestMemory>
+          <resourceLimitCpu></resourceLimitCpu>
+          <resourceLimitMemory></resourceLimitMemory>
+          <envVars/>
+        </org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
+      </containers>
+      <envVars/>
+      <annotations/>
+      <imagePullSecrets/>
+      <nodeProperties/>
+    </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
+----
+
+[NOTE]
+====
+If you log into the Jenkins console and make further changes to the Pod Template
+configuration after the Pod Template is created, and the OpenShift Sync plug-in
+detects that the ConfigMap has changed, it will replace the Pod Template and
+overwrite those configuration changes. No merging of configuration will occur.
+====
+
+After startup, the {product-title} Sync plug-in monitors the API server of
```

Comment:
the {product-title} Sync plug-in should be ` the Openshift Sync plug-in` as above

Reply:
I went ahead and updated the reference, but to clarify, it shouldn't be `OpenShift Container Platform Sync plug-in`?

Reply:
Should be openshift sync plug-in,  see https://github.com/openshift/jenkins/blob/master/2/contrib/openshift/base-plugins.txt#L12